Utils/JSONSchema/infer.h

```diff
@@ -338,6 +338,10 @@ inline Schema RecursivelyInferSchema(const rapidjson::Value& value) {
     }
   } else if (value.IsString()) {
     return String(std::string(value.GetString(), value.GetStringLength()));
+#if 0  // Hack. -- D.K.
```

Comment:
Right now integers are not supported at all by this tool. I looked at the distribution of their values via this trick.